apps/forklift/test/unit/integration/event_handling_test.exs

```diff
@@ -23,6 +23,12 @@ defmodule Forklift.Integration.EventHandlingTest do
       Brook.Event.send(instance_name(), dataset_update(), :author, dataset)
       assert_receive table: ^table_name, schema: ^schema
     end
+
+    test "does not create table for non-ingestible dataset" do
+      expect(MockTable, :init, 0, fn _ -> :ok end)
+      dataset = TDG.create_dataset(%{technical: %{sourceType: "remote"}})
```

Comment:
Is `ingestible` something we should define somewhere central? Or is it different for each app? LGTM and I don't think we should address that here. However, it has come up before.

Reply:
It's the same across the platform, so you're probably right.

Reply:
Good.